docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: "3.7"
```

Comment:
Travis CI has a default `docker-compose` that allows up to `3.4`, compare https://github.com/libero/sample-configuration/blob/master/docker-compose.yml#
A newer version requires to update `docker-compose` on every build, but it doesn't look like there is syntax in here that requires a very new version

Reply:
updated